osclib/stagingapi.py

```diff
@@ -1014,14 +1024,28 @@ def package_version(self, project, package):
                 pass
         return version
 
-    def load_file_content(self, project, package, filename):
+    def guess_binary_version(self, project, package, repository='standard', arch='x86_64'):
+        """
+        Return the most probable version of the package in question
+        """
+        buildinfo = self.load_file_content(project, package, '_buildenv', binary=True, repository=repository, arch=arch)
```

Comment:
On Mon, 2015-11-30 at 01:55 -0800, Stephan Kulow wrote:
> 
> You have to know your api ;)
> osc api /build/openSUSE:Factory/totest/x86_64/_repository/openSUSE-
> release.rpm?view=fileinfo
> —
Thanks... I simplified using this API.